### DISCUSSION TITLE
Reakit/button vs native button

### DISCUSSION THREAD

**Discussion by VladSez** [ariakit/ariakit#762] [Questions] [answered]
What are the advantages of using reakit/Button instead of native button?
Great library, by the way😄

**Comment by diegohaz** [comment, marked as answer]
You get consistent behavior across browsers:
https://github.com/reakit/reakit/blob/9a4b2f844d7bb9c87e1ac359bbeacddf93488500/packages/reakit/src/Tabbable/Tabbable.ts#L50-L109
